src/buffered-process.js

```diff
@@ -189,12 +189,13 @@ class BufferedProcess {
       output += data
     })
     wmicProcess.stdout.on('close', () => {
-      const pidsToKill = output.split(/\s+/)
-        .filter((pid) => /^\d+$/.test(pid))
-        .map((pid) => parseInt(pid))
-        .filter((pid) => pid !== parentPid && pid > 0 && pid < Infinity)
+      for (let pid of output.split(/\s+/)) {
+        if (!isFinite(pid)) continue
+
+        pid = parseInt(pid, 10)
+
+        if (!pid || pid === parentPid || pid < 0 || pid === Infinity) continue
```

Comment:
`pid === Infinity` will be never hit because `parseInt` won't return `Infinity`, otherwise should sufficiently cover rest of the possible outputs of `parseInt`.

Reply:
Yes, check for `Infinity` is needless in this case.